### PR TITLE
feat & fix: RESKIN Cookies

### PR DESCRIPTION
**Pull request by nesdc88**
## Ticket
https://trello.com/c/1ElcR7nr/591-reskin-cookies

## Description
RESKIN Cookies

## What solved

- [X] Should update the green color to Sky blue #504DFF dark/light mode
- [X] Should update the background, dark/light mode
- [X] Should update the hover status of the Configure button
- [X] Should update the green color to Sky blue #504DFF dark/light mode (404 page)
- [X] The pop up window should display the full information (fix)

## Checklist:

- [X] I have performed a self-review of my own code
- [X] I have performed a self-review of my own chromatic changes
- [X] I have commented on my code, particularly in hard-to-understand areas
- [X] I have checked my code and corrected any misspellings
- [X] I have removed any unnecessary console messages
- [X] I have removed any commented code
- [X] I have checked that there are no buggy stories in Storybook

### CODE REVIEW

File: src/core/context/MainWrapper.tsx [hunk 1]
```diff
@@ -61,24 +60,25 @@ const MainWrapper = ({ children }: { children: ReactNode }) => {
...
   top: 0,
   width: '100%',
   zIndex: zIndexEnum.OVERLAY_MOBILE_TOOLTIP,
   height: 'calc(100vh - 282px)',
   background: 'rgba(52, 52, 66, 0.1)',
-  backdropFilter: isLight ? 'blur(2px)' : 'blur(4px)',
-  [lightTheme.breakpoints.between('mobile_375', 'table_834')]: {
+  backdropFilter: theme.palette.isLight ? 'blur(2px)' : 'blur(4px)',
+
+  [theme.breakpoints.down('table_834')]: {
```
Comment (YasielCabrera) [inline src/core/context/MainWrapper.tsx:72]: You should use `tablet_768` as `table_834` is a legacy breakpoint that shouldn't be used anymore. Same with `desktop_1194`

File: src/stories/components/404/NotFound404.tsx [hunk 2]
```diff
@@ -1,36 +1,33 @@
...
+import type { FC } from 'react';
 
-const NotFound404: React.FC = () => {
+const NotFound404: FC = () => {
   const router = useRouter();
-  const phoneLess = useMediaQuery(lightTheme.breakpoints.down('mobile_375'));
-  const isMobile = useMediaQuery(lightTheme.breakpoints.between('mobile_375', 'table_834'));
-  const { isLight } = useThemeContext();
+  const theme = useTheme();
+  const isMobile = useMediaQuery((theme: Theme) => theme.breakpoints.down('table_834'));
```
Comment (YasielCabrera) [inline src/stories/components/404/NotFound404.tsx:17]: Same than above, this is a legacy breakpoint

File: src/stories/components/404/NotFound404.tsx [hunk 3]
```diff
@@ -87,17 +62,18 @@ const NotFound404: React.FC = () => {
...
 
-const Wrapper = styled.div({
+const Wrapper = styled('div')(({ theme }) => ({
   width: '100%',
   height: '100%',
   marginTop: '132px',
   paddingBottom: '128px',
-  [lightTheme.breakpoints.up('table_834')]: {
+
+  [theme.breakpoints.up('table_834')]: {
```
Comment (YasielCabrera) [inline src/stories/components/404/NotFound404.tsx:71]: Same than above, this is a legacy breakpoint

File: src/stories/components/404/NotFound404.tsx [hunk 4]
```diff
@@ -109,100 +85,101 @@ const ImageContainer = styled.div<{ isLight?: boolean }>(({ isLight }) => ({
...
   borderRadius: '20px',
 
-  [lightTheme.breakpoints.between('table_834', 'desktop_1194')]: {
+  [theme.breakpoints.between('table_834', 'desktop_1194')]: {
     paddingTop: '128px',
     maxWidth: '769px',
     margin: '0 auto',
   },
-  [lightTheme.breakpoints.between('desktop_1194', 'desktop_1280')]: {
+  [theme.breakpoints.between('desktop_1194', 'desktop_1280')]: {
```
Comment (YasielCabrera) [inline src/stories/components/404/NotFound404.tsx:100]: Same than above, these are a legacy breakpoints

File: src/stories/components/404/NotFound404.tsx [hunk 5]
```diff
@@ -109,100 +85,101 @@ const ImageContainer = styled.div<{ isLight?: boolean }>(({ isLight }) => ({
...
   position: 'relative',
   display: 'block',
   width: '279px',
   height: '164px',
   marginBottom: '64px',
   marginLeft: '32px',
   marginRight: '32px',
-  [lightTheme.breakpoints.between('table_834', 'desktop_1194')]: {
+
+  [theme.breakpoints.between('table_834', 'desktop_1194')]: {
```
Comment (YasielCabrera) [inline src/stories/components/404/NotFound404.tsx:137]: legacy breakpoint here

File: src/stories/components/404/NotFound404.tsx [hunk 6]
```diff
@@ -109,100 +85,101 @@ const ImageContainer = styled.div<{ isLight?: boolean }>(({ isLight }) => ({
...
     marginBottom: '160px',
     width: '580px',
     height: '340px',
   },
-  [lightTheme.breakpoints.up('desktop_1194')]: {
+  [theme.breakpoints.up('desktop_1194')]: {
```
Comment (YasielCabrera) [inline src/stories/components/404/NotFound404.tsx:142]: legacy breakpoint here

File: src/stories/components/404/NotFound404.tsx [hunk 7]
```diff
@@ -109,100 +85,101 @@ const ImageContainer = styled.div<{ isLight?: boolean }>(({ isLight }) => ({
...
   display: 'flex',
   flexDirection: 'column',
   justifyContent: 'center',
   alignItems: 'center',
   paddingRight: '32px',
   paddingLeft: '32px',
   maxWidth: '343px',
-  [lightTheme.breakpoints.up('table_834')]: {
+
+  [theme.breakpoints.up('table_834')]: {
```
Comment (YasielCabrera) [inline src/stories/components/404/NotFound404.tsx:158]: legacy breakpoint here

File: src/stories/components/404/NotFound404.tsx [hunk 8]
```diff
@@ -109,100 +85,101 @@ const ImageContainer = styled.div<{ isLight?: boolean }>(({ isLight }) => ({
...
+  letterSpacing: '0.4px',
+  color: theme.palette.isLight ? '#787A9B' : '#D2D4EF',
+  textAlign: 'center',
+  marginBottom: '24px',
 
-const TextUps = styled(Typography, { shouldForwardProp: (prop) => prop !== 'isLight' })<{ isLight?: boolean }>(
-  ({ isLight }) => ({
-    fontFamily: 'Inter, sans-serif',
-    fontStyle: 'normal',
+  [theme.breakpoints.up('table_834')]: {
```
Comment (YasielCabrera) [inline src/stories/components/404/NotFound404.tsx:174]: legacy breakpoint here

File: src/stories/components/404/NotFound404.tsx [hunk 9]
```diff
@@ -212,19 +189,47 @@ const TextDescription = styled(Typography)({
   letterSpacing: '0.4px',
   color: '#ADAFD4',
   marginBottom: '64px',
-  [lightTheme.breakpoints.up('table_834')]: {
+
+  [theme.breakpoints.up('table_834')]: {
```
Comment (YasielCabrera) [inline src/stories/components/404/NotFound404.tsx:193]: legacy breakpoint here

File: src/stories/components/404/NotFound404.tsx [hunk 10]
```diff
@@ -212,19 +189,47 @@ const TextDescription = styled(Typography)({
...
-const ContainerButton = styled.div({
+const ContainerButton = styled('div')(({ theme }) => ({
   paddingBottom: '83px',
-  [lightTheme.breakpoints.between('table_834', 'desktop_1194')]: {
+
+  [theme.breakpoints.between('table_834', 'desktop_1194')]: {
     paddingBottom: '156px',
   },
-  [lightTheme.breakpoints.up('desktop_1194')]: {
+  [theme.breakpoints.up('desktop_1194')]: {
```
Comment (YasielCabrera) [inline src/stories/components/404/NotFound404.tsx:206]: legacy breakpoint here

File: src/stories/components/404/NotFound404.tsx [hunk 11]
```diff
@@ -212,19 +189,47 @@ const TextDescription = styled(Typography)({
...
   },
-});
+}));
 
-const ContainerButton = styled.div({
+const ContainerButton = styled('div')(({ theme }) => ({
   paddingBottom: '83px',
-  [lightTheme.breakpoints.between('table_834', 'desktop_1194')]: {
+
+  [theme.breakpoints.between('table_834', 'desktop_1194')]: {
```
Comment (YasielCabrera) [inline src/stories/components/404/NotFound404.tsx:203]: legacy breakpoint here

File: src/stories/components/CookiesPolicyBanner/CookiesPolicyBanner.tsx [hunk 12]
```diff
@@ -16,172 +13,196 @@ interface Props {
...
+const Container = styled('div')(({ theme }) => ({
+  background: theme.palette.isLight ? '#FFFFFF' : '#10191F',
   borderRadius: '6px',
   padding: '40px 24px',
   height: '458px',
-  boxShadow: !isLight ? '0px -15px 35px 10px rgba(0, 27, 141, 0.15)' : 'none',
-  [lightTheme.breakpoints.between('table_834', 'desktop_1194')]: {
+  boxShadow: !theme.palette.isLight ? '0px -15px 35px 10px rgba(0, 27, 141, 0.15)' : 'none',
+
+  [theme.breakpoints.between('table_834', 'desktop_1194')]: {
```
Comment (YasielCabrera) [inline src/stories/components/CookiesPolicyBanner/CookiesPolicyBanner.tsx:74]: legacy breakpoint here

File: src/stories/components/CookiesPolicyBanner/CookiesPolicyBanner.tsx [hunk 13]
```diff
@@ -16,172 +13,196 @@ interface Props {
...
     padding: '40px 63px',
   },
-  [lightTheme.breakpoints.between('desktop_1194', 'desktop_1280')]: {
+  [theme.breakpoints.between('desktop_1194', 'desktop_1280')]: {
```
Comment (YasielCabrera) [inline src/stories/components/CookiesPolicyBanner/CookiesPolicyBanner.tsx:77]: legacy breakpoint here

File: src/stories/components/CookiesPolicyBanner/CookiesPolicyBanner.tsx [hunk 14]
```diff
@@ -16,172 +13,196 @@ interface Props {
...
-  [lightTheme.breakpoints.between('desktop_1440', 'desktop_1920')]: {
+  [theme.breakpoints.between('desktop_1440', 'desktop_1920')]: {
     padding: '40px 366px',
   },
-  [lightTheme.breakpoints.up('desktop_1920')]: {
+  [theme.breakpoints.up('desktop_1920')]: {
     padding: '40px 606px',
   },
-  [lightTheme.breakpoints.up('table_834')]: {
+  [theme.breakpoints.up('table_834')]: {
```
Comment (YasielCabrera) [inline src/stories/components/CookiesPolicyBanner/CookiesPolicyBanner.tsx:89]: legacy breakpoint here

File: src/stories/components/CookiesPolicyBanner/CookiesPolicyBanner.tsx [hunk 15]
```diff
@@ -16,172 +13,196 @@ interface Props {
...
 
-const ContainerData = styled.div({
+const ContainerData = styled('div')(({ theme }) => ({
   display: 'flex',
   flexDirection: 'column',
   maxWidth: '375px',
   margin: '0px auto',
-  [lightTheme.breakpoints.up('table_834')]: {
+
+  [theme.breakpoints.up('table_834')]: {
```
Comment (YasielCabrera) [inline src/stories/components/CookiesPolicyBanner/CookiesPolicyBanner.tsx:100]: legacy breakpoint here

File: src/stories/components/CookiesPolicyBanner/CookiesPolicyBanner.tsx [hunk 16]
```diff
@@ -16,172 +13,196 @@ interface Props {
...
+
+const ContainerCheckBox = styled('div')(({ theme }) => ({
   display: 'flex',
   flexDirection: 'row',
   gap: '34px',
   margin: '0 auto',
   marginTop: '32px',
-  [lightTheme.breakpoints.up('table_834')]: {
+
+  [theme.breakpoints.up('table_834')]: {
```
Comment (YasielCabrera) [inline src/stories/components/CookiesPolicyBanner/CookiesPolicyBanner.tsx:129]: legacy breakpoint here

File: src/stories/components/CookiesPolicyBanner/CookiesPolicyBanner.tsx [hunk 17]
```diff
@@ -16,172 +13,196 @@ interface Props {
...
+const ContainerButton = styled('div')(({ theme }) => ({
   display: 'flex',
   flexDirection: 'column',
   paddingLeft: '22px',
   paddingRight: '22px',
   marginTop: '32px',
   alignItems: 'center',
-  [lightTheme.breakpoints.up('table_834')]: {
+
+  [theme.breakpoints.up('table_834')]: {
```
Comment (YasielCabrera) [inline src/stories/components/CookiesPolicyBanner/CookiesPolicyBanner.tsx:142]: legacy breakpoint here

File: src/stories/components/CookiesPolicyBanner/CookiesPolicyBanner.tsx [hunk 18]
```diff
@@ -16,172 +13,196 @@ interface Props {
...
+    },
+  },
   '& > div': {
     fontSize: '16px',
     lineHeight: '19px',
+    color: theme.palette.isLight ? theme.palette.colors.gray[50] : theme.palette.colors.slate[50],
   },
-  [lightTheme.breakpoints.up('table_834')]: {
+
+  [theme.breakpoints.up('table_834')]: {
```
Comment (YasielCabrera) [inline src/stories/components/CookiesPolicyBanner/CookiesPolicyBanner.tsx:176]: legacy breakpoint here

File: src/stories/components/CookiesPolicyBanner/CookiesPolicyBanner.tsx [hunk 19]
```diff
@@ -16,172 +13,196 @@ interface Props {
...
+  backgroundColor: theme.palette.isLight ? theme.palette.colors.gray[200] : theme.palette.colors.slate[500],
+  border: 'none',
   '& > div': {
     fontSize: '16px',
     lineHeight: '19px',
+    color: theme.palette.isLight ? theme.palette.colors.gray[600] : theme.palette.colors.slate[100],
   },
 
-  [lightTheme.breakpoints.up('table_834')]: {
+  [theme.breakpoints.up('table_834')]: {
```
Comment (YasielCabrera) [inline src/stories/components/CookiesPolicyBanner/CookiesPolicyBanner.tsx:193]: legacy breakpoint here

File: src/stories/containers/CookiesPolicy/CookiesPolicyContainer.tsx [hunk 20]
```diff
@@ -55,129 +52,147 @@ const CookiesPolicyContainer = () => {
...
-  [lightTheme.breakpoints.between('desktop_1194', 'desktop_1280')]: {
+  [theme.breakpoints.between('desktop_1194', 'desktop_1280')]: {
     maxWidth: '1066px',
   },
-  [lightTheme.breakpoints.between('table_834', 'desktop_1194')]: {
+  [theme.breakpoints.between('table_834', 'desktop_1194')]: {
     maxWidth: '770px',
   },
-  [lightTheme.breakpoints.between('mobile_375', 'table_834')]: {
+  [theme.breakpoints.down('table_834')]: {
```
Comment (YasielCabrera) [inline src/stories/containers/CookiesPolicy/CookiesPolicyContainer.tsx:90]: legacy breakpoint here

File: src/stories/containers/CookiesPolicy/CookiesPolicyContainer.tsx [hunk 21]
```diff
@@ -55,129 +52,147 @@ const CookiesPolicyContainer = () => {
...
-  textAlign: 'left',
-  [lightTheme.breakpoints.between('table_834', 'desktop_1194')]: {
-    textAlign: 'left',
-  },
-  [lightTheme.breakpoints.between('desktop_1194', 'desktop_1280')]: {
-    textAlign: 'left',
+  color: theme.palette.isLight ? theme.palette.colors.gray[900] : theme.palette.colors.gray[50],
+  marginBottom: 16,
+
+  [theme.breakpoints.between('table_834', 'desktop_1194')]: {
```
Comment (YasielCabrera) [inline src/stories/containers/CookiesPolicy/CookiesPolicyContainer.tsx:107]: legacy breakpoint here

File: src/stories/containers/CookiesPolicy/CookiesPolicyContainer.tsx [hunk 22]
```diff
@@ -55,129 +52,147 @@ const CookiesPolicyContainer = () => {
...
   fontStyle: 'normal',
-  fontWeight: 700,
+  fontWeight: 600,
   fontSize: '16px',
-  lineHeight: '19px',
-  color: isLight ? '#231536' : '#D2D4EF',
+  lineHeight: '24px',
+  color: theme.palette.isLight ? theme.palette.colors.gray[900] : theme.palette.colors.gray[50],
+
+  [theme.breakpoints.up('table_834')]: {
```
Comment (YasielCabrera) [inline src/stories/containers/CookiesPolicy/CookiesPolicyContainer.tsx:121]: legacy breakpoint here

File: src/stories/containers/CookiesPolicy/CookiesPolicyContainer.tsx [hunk 23]
```diff
@@ -55,129 +52,147 @@ const CookiesPolicyContainer = () => {
...
 
-const ContainerButton = styled.div({
+const ContainerButton = styled('div')(({ theme }) => ({
   display: 'flex',
   flexDirection: 'row',
   justifyContent: 'center',
   marginTop: '64px',
-  [lightTheme.breakpoints.between('mobile_375', 'table_834')]: {
+
+  [theme.breakpoints.down('table_834')]: {
```
Comment (YasielCabrera) [inline src/stories/containers/CookiesPolicy/CookiesPolicyContainer.tsx:144]: legacy breakpoint here

File: src/stories/containers/CookiesPolicy/CookiesPolicyContainer.tsx [hunk 24]
```diff
@@ -55,129 +52,147 @@ const CookiesPolicyContainer = () => {
...
   fontStyle: 'normal',
   fontWeight: 400,
-  fontSize: '16px',
-  lineHeight: '22px',
-  color: isLight ? '#231536' : '#D2D4EF',
+  fontSize: 14,
+  lineHeight: '24px',
+  color: theme.palette.isLight ? theme.palette.colors.gray[600] : theme.palette.colors.gray[50],
+
+  [theme.breakpoints.up('desktop_1194')]: {
```
Comment (YasielCabrera) [inline src/stories/containers/CookiesPolicy/CookiesPolicyContainer.tsx:157]: legacy breakpoint here

File: src/stories/containers/CookiesPolicy/CookiesPolicyContainer.tsx [hunk 25]
```diff
@@ -55,129 +52,147 @@ const CookiesPolicyContainer = () => {
...
-  fontWeight: 400,
-  fontSize: '16px',
-  lineHeight: '24px',
-  color: isLight ? '#231536' : '#D2D4EF',
+  fontWeight: 600,
+  fontSize: 14,
+  lineHeight: '22px',
+  color: theme.palette.isLight ? theme.palette.colors.gray[600] : theme.palette.colors.gray[50],
+
+  [theme.breakpoints.up('desktop_1194')]: {
```
Comment (YasielCabrera) [inline src/stories/containers/CookiesPolicy/CookiesPolicyContainer.tsx:170]: legacy breakpoint here

**Review by YasielCabrera — CHANGES_REQUESTED** [review]

**Review by alejandrocabriales — APPROVED** [review]
LGTM

File: src/core/context/MainWrapper.tsx (hunk 1 above)
Comment (nesdc88) [reply src/core/context/MainWrapper.tsx:72]: Done

File: src/stories/components/404/NotFound404.tsx (hunk 2 above)
Comment (nesdc88) [reply src/stories/components/404/NotFound404.tsx:17]: Done

File: src/stories/components/404/NotFound404.tsx (hunk 3 above)
Comment (nesdc88) [reply src/stories/components/404/NotFound404.tsx:71]: Done

File: src/stories/components/404/NotFound404.tsx (hunk 4 above)
Comment (nesdc88) [reply src/stories/components/404/NotFound404.tsx:100]: Done

File: src/stories/components/404/NotFound404.tsx (hunk 5 above)
Comment (nesdc88) [reply src/stories/components/404/NotFound404.tsx:137]: Done

File: src/stories/containers/CookiesPolicy/CookiesPolicyContainer.tsx (hunk 21 above)
Comment (nesdc88) [reply src/stories/containers/CookiesPolicy/CookiesPolicyContainer.tsx:107]: Done

File: src/stories/containers/CookiesPolicy/CookiesPolicyContainer.tsx (hunk 22 above)
Comment (nesdc88) [reply src/stories/containers/CookiesPolicy/CookiesPolicyContainer.tsx:121]: Done

File: src/stories/containers/CookiesPolicy/CookiesPolicyContainer.tsx (hunk 23 above)
Comment (nesdc88) [reply src/stories/containers/CookiesPolicy/CookiesPolicyContainer.tsx:144]: Done

File: src/stories/containers/CookiesPolicy/CookiesPolicyContainer.tsx (hunk 24 above)
Comment (nesdc88) [reply src/stories/containers/CookiesPolicy/CookiesPolicyContainer.tsx:157]: Done

File: src/stories/containers/CookiesPolicy/CookiesPolicyContainer.tsx (hunk 25 above)
Comment (nesdc88) [reply src/stories/containers/CookiesPolicy/CookiesPolicyContainer.tsx:170]: Done

File: src/stories/components/404/NotFound404.tsx (hunk 6 above)
Comment (nesdc88) [reply src/stories/components/404/NotFound404.tsx:142]: Done

File: src/stories/components/404/NotFound404.tsx (hunk 7 above)
Comment (nesdc88) [reply src/stories/components/404/NotFound404.tsx:158]: Done

File: src/stories/components/404/NotFound404.tsx (hunk 8 above)
Comment (nesdc88) [reply src/stories/components/404/NotFound404.tsx:174]: Done

File: src/stories/components/404/NotFound404.tsx (hunk 10 above)
Comment (nesdc88) [reply src/stories/components/404/NotFound404.tsx:206]: Done

File: src/stories/components/404/NotFound404.tsx (hunk 11 above)
Comment (nesdc88) [reply src/stories/components/404/NotFound404.tsx:203]: Done

File: src/stories/components/CookiesPolicyBanner/CookiesPolicyBanner.tsx (hunk 12 above)
Comment (nesdc88) [reply src/stories/components/CookiesPolicyBanner/CookiesPolicyBanner.tsx:74]: Done

File: src/stories/components/CookiesPolicyBanner/CookiesPolicyBanner.tsx (hunk 13 above)
Comment (nesdc88) [reply src/stories/components/CookiesPolicyBanner/CookiesPolicyBanner.tsx:77]: Done

File: src/stories/components/CookiesPolicyBanner/CookiesPolicyBanner.tsx (hunk 14 above)
Comment (nesdc88) [reply src/stories/components/CookiesPolicyBanner/CookiesPolicyBanner.tsx:89]: Done

File: src/stories/components/CookiesPolicyBanner/CookiesPolicyBanner.tsx (hunk 15 above)
Comment (nesdc88) [reply src/stories/components/CookiesPolicyBanner/CookiesPolicyBanner.tsx:100]: Done

File: src/stories/components/CookiesPolicyBanner/CookiesPolicyBanner.tsx (hunk 16 above)
Comment (nesdc88) [reply src/stories/components/CookiesPolicyBanner/CookiesPolicyBanner.tsx:129]: Done

File: src/stories/components/CookiesPolicyBanner/CookiesPolicyBanner.tsx (hunk 17 above)
Comment (nesdc88) [reply src/stories/components/CookiesPolicyBanner/CookiesPolicyBanner.tsx:142]: Done

File: src/stories/components/CookiesPolicyBanner/CookiesPolicyBanner.tsx (hunk 18 above)
Comment (nesdc88) [reply src/stories/components/CookiesPolicyBanner/CookiesPolicyBanner.tsx:176]: Done

File: src/stories/components/CookiesPolicyBanner/CookiesPolicyBanner.tsx (hunk 19 above)
Comment (nesdc88) [reply src/stories/components/CookiesPolicyBanner/CookiesPolicyBanner.tsx:193]: Done

**Pull request merged**